test/e2e/setup_test.go

```diff
@@ -186,6 +186,16 @@ var (
 		ignoreConditionLTTAndMessageFields,
 		cmpopts.EquateEmpty(),
 	}
+
+	safeDeploymentCRPStatusCmpOptions = cmp.Options{
+		cmpopts.SortSlices(lessFuncCondition),
+		cmpopts.SortSlices(lessFuncPlacementStatus),
```

Comment:
AFAIk, this sorts the resourceplacementStatus by clusterName but we are not sure which cluster gets rolled out first (in theory).  Wouldn't this be a random test case?

Reply:
but if we don't use this function, how can they compare then?